model/workflow.go

```diff
@@ -15,8 +15,12 @@
 package model
 
 import (
+	"bytes"
 	"encoding/json"
 	"fmt"
+	"github.com/go-playground/validator/v10"
+	val "github.com/serverlessworkflow/sdk-go/v2/validator"
```

Comment:
should we organize the imports by it's category？

Reply:
it was edited by intellij, we can later define a code style and adjust it.